### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Sargis** [Makuna/Rfid134#19] [Q&A]
Get get following error

error: 'Serial1' was not declared in this scope
 Rfid134<HardwareSerial, RfidNotify> rfid(Serial1);
 
 
 how can I fix this?

**Comment by Makuna** [comment]
(issues is not the area to ask questions, moved to discussions)

You need to replace it with an available serial object.  Whatever platform of Arduino you are using doesn't seem to have two Serial devices (Serial and Serial1).

You may have to use SoftwareSerial library.

**Reply by Sargis** [reply]
Sorry Makuna

Thank you for responding and offering to help me.
